### ISSUE TITLE
web editor not importing preloaded project

### ISSUE THREAD

**Issue by irisgames404** [godotengine/godot#74744]
### Godot version

4.0.stable

### System information

windows 10

### Issue description

When you add a project into the preload section of the web editor [(link)](https://editor.godotengine.org/releases/4.0.stable/godot.editor.html)
![image](https://user-images.githubusercontent.com/114032547/224430662-bf814be6-877b-404c-bbf4-d1dea112891c.png)
it accepts the file but wont load it or realize it has been loaded
[(link to video)](https://drive.google.com/file/d/1eVbse4GxRxz23phFQoDciwHvVuXWrVdw/view?usp=sharing)

### Steps to reproduce

try import a project into the preload of the web editor

### Minimal reproduction project

[demo.zip](https://github.com/godotengine/godot/files/10946226/demo.zip)

**Comment by akien-mga** [comment]
It actually loads it, but the logic which is supposed to install it automatically seems broken.

You'll find it in `/tmp/preload.zip` in the web filesystem, so if you press "Import" you can actually import `/tmp/preload.zip` in `/home/web_user/`. But in the past this used to be automatic, and that should be restored.

**Comment by manuq** [comment]
Yes I confirm, now the user has to navigate to `/tmp` manually, making the import a bit cumbersome.
